spec/ttml1.xml

```diff
@@ -4300,29 +4300,55 @@ one column in width.</p>
 <p>A glyph's EM square is conventionally defined as the EM square of the font that contains the glyph. That is,
 glyphs do not have an EM square that is distinct from their font's EM square.</p>
 </note>
+
 <p>If horizontal and vertical sizes are expressed independently, then the
 units of the <loc href="#style-value-length">&lt;length&gt;</loc> values must be the same.</p>
 <p>The <loc href="#style-value-length">&lt;length&gt;</loc> value(s) used to express font size must be non-negative.</p>
+
+<p>Relative <loc href="#style-value-length">&lt;length&gt;</loc> values
+that appear in this attribute, i.e., values expressed in percentage (<code>%</code>), cell (<code>c</code>), or EM (<code>em</code>) units,
+are resolved in relation to a referenced 2-dimensional size value consisting of two components, a width component and
+a height component.</p>
+
+
+<p>When a single relative <loc href="#style-value-length">&lt;length&gt;</loc> value is specified, then
+this <loc href="#style-value-length">&lt;length&gt;</loc> is resolved in terms of the height component of the referenced value;
+when two relative
+<loc href="#style-value-length">&lt;length&gt;</loc> values are specified,
+the first <loc href="#style-value-length">&lt;length&gt;</loc> is resolved in terms of the width component of the referenced value and
+the second <loc href="#style-value-length">&lt;length&gt;</loc> is resolved in terms of the height component of the referenced value.</p>
+
+<p>If the relative unit is a percentage (<code>%</code>), then the referenced value is the <emph>Computed Cell Size</emph> if specified on the <loc href="#layout-vocabulary-region"><el>region</el></loc> element; or the parent element's computed font size otherwise.</p>
+
+<p>If the relative unit is cell (<code>c</code>), then the referenced value is the
+<emph>Computed Cell Size</emph>.</p>
+
+<note role="example">
+<p>For example, consider a paragraph (<code>p</code>) element <code>P</code>.
+If the <emph>Computed Cell Size</emph> is (24px,36px), and if
```

Comment:
Same comment as https://github.com/w3c/ttml2/pull/336/files#r157899921 :

As per the discussion in https://github.com/w3c/ttml2/issues/200#issuecomment-305547161 the sizes here are in logical pixels - in fact they are dimensionless until multiplied by the size of the pixel unit, whatever that turns out to be. Consider adding a parenthetical _(in logical pixels)_ to avoid confusion with "real" pixels.